.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Fix checkstyle warnings for tests
+57f261a6c4326ed63fcf908bb0a5dac28f247d86
```

Comment:
Keep in mind that if you merge via mergify/ready-to-merge label this well get rebased and the hash will change.

Reply:
yep yep, thx